pages.nl/linux/apt.md

```diff
@@ -0,0 +1,38 @@
+# apt
+
+> Hulpprogramma voor pakketbeheer voor op Debian gebaseerde distributies.
+> Aanbevolen vervanging voor `apt-get` bij interactief gebruik in Ubuntu versie 16.04 en later.
+> Voor gelijkwaardige commando's in andere pakket managers, zie <https://wiki.archlinux.org/title/Pacman/Rosetta>.
+> Meer informatie: <https://manpages.debian.org/latest/apt/apt.8.html>.
+
+- Werk de lijst van beschikbare pakketten en versies bij (het wordt aanbevolen dit uit te voeren voor elk ander `apt` commando):
+
+`sudo apt update`
+
+- Zoek naar een bepaald pakket:
+
+`apt search {{package}}`
```

Comment:
```suggestion
- Zoek naar een specifiek pakket:

`apt search {{pakket}}`
```

Optional fix (error source: original page): broken description: `specific` expected.
Note: translation source: Search for a specific package.

Reply:
Thanks for the review! Fixed in the latest commit.